components/layout/Wrapper.tsx

```diff
@@ -0,0 +1,17 @@
+import stls from '@/styles/components/layout/Wrapper.module.sass'
+import { TypeChildren, TypeClassNames } from '@/types/index'
+import cn from 'classnames'
+import { getClassNames } from 'helperts/index'
```

Comment:
there is a typo error & imports should be absolute and start with `@/`

```suggestion
import { getClassNames } from '@/helpers/index'
```